@xen-orchestra/lite/src/components/pool/dashboard/cpuUsage/VmsCpuUsage.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div v-if="data.length !== 0">
```

Comment:
Same question here about moving the `div` inside `UsageBar` and let it handle `v-if`